### ISSUE TITLE
Installed pacman packages with numbers are reinstalled

### ISSUE THREAD

**Issue by unai-ndz** [Fizzadar/pyinfra#504]
**Describe the bug**
The regex in host.fact.pacman_packages does not allow numbers in the names, this effectively "removes" them from the list of installed packages.
[The arch package database](https://archlinux.org/packages/) shows packages with numbers.

**To Reproduce**
Running this will install m4 even if it's already installed

```python
pacman.packages(
    name='Install m4',
    sudo=True,
    packages=[
        'm4',
    ],
)
```

**Expected behavior**
The above should not run pacman, just return unchanged.

**Comment by Fizzadar** [comment]
I've fixed this in https://github.com/Fizzadar/pyinfra/commit/16786b2df690891d11a052fca84adc13c58a7d9f!

**Comment by Fizzadar** [comment]
Now released in `1.3.1`!
